src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Tree/Dependencies/Subscriptions/ConfiguredDependencyFilterBlock.cs

```diff
@@ -0,0 +1,73 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Threading.Tasks.Dataflow;
+using Microsoft.VisualStudio.ProjectSystem.Utilities;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Tree.Dependencies.Subscriptions;
+
+internal static class ConfiguredDependencyFilterBlock
+{
+    public static ISourceBlock<IProjectVersionedValue<T>> TransformSource<T>(ISourceBlock<IProjectVersionedValue<T>> source, DisposableBag disposables, string nameFormat)
```

Comment:
This would benefit from a doc comment explaining when and why it should be used.

Reply:
The doc comment is on the inner type, which was an outer type in an earlier revision of this PR. I'll pull it up and update the doc.